src/player_activity.h

```diff
@@ -25,13 +27,14 @@ class activity_type;
 class monster;
 class translation;
 
-using activity_id = string_id<activity_type>;
-
 class player_activity
 {
     private:
         activity_id type;
+        cata::clone_ptr<activity_actor> actor;
```

Comment:
Does `player_activity` need to be copyable?  Could this just be a `unique_ptr`?

Reply:
It may not be strictly necessary, but it seemed easier than tracking down all the places we are currently doing copy construction and copy assignment to add an `std::move()` thanks to the `cata::clone_ptr` type you added.  I guess I didn't think about it too deeply but don't see a big drawback in keeping it copyable.

Reply:
Yeah, if it's already being copied sometimes then that's probably fine.